rancher/install.go

```diff
@@ -125,9 +125,15 @@ func DeployRancherManager(hostname, channel, version, headVersion, ca, proxy str
 	}
 
 	// Use Rancher Manager behind proxy
+	// Get the proxyHost if given
+	var proxyHost string
+	proxyHost = "http://172.17.0.1:3128"
```

Comment:
This 2 lines could be replaced by `proxyHost := "http://172.17.0.1:3128"` I think.

Reply:
Done.